src/defaultOptions.js

```diff
@@ -58,12 +62,10 @@ const everyMonthOptions = {
         info:
           'Monthly gifts help us focus on our mission and long-term impact',
         levels: [
-          { amount: '5' },
-          { amount: '10' },
-          { amount: '20' },
+          { amount: '25' },
           { amount: '50' },
           { amount: '100' },
-          { amount: '200' },
+          { amount: '500' },
```

Comment:
Are these for the monthly donation levels? A bit confused as to why they are under `en` section and why this code does not match the existing 25/50/100 options, do they need to be repeated twice could we remove this section?

Reply:
I agree that it's confusing, we should make another pass at the options format in a major version. the way it is now is counterintuitive, if you don't provide everything in i18n then it'll just null dereference and crash, it's an emergent property of how the design evolved; we can streamline the format a lot but for now just to get this going let's go with it

Reply:
Inside `monthly` we have the generic and common part of the config. Inside `i18n` we have the labels and descriptions that corresponds to each language. The amount prop act as a key to link them.  
For the minimum default values, isn't neccessary anymore provide texts, so they seems useless at this point

Reply:
I'll make a proposal later but basically I think we shouldn't be using amounts as an ID and then having to ad-hoc make sure they're matching each other in separate parts of the config; I'd rather that everything related to an amount is centralized in one place, and anywhere where you can config a string, you can alternatively provide an object from language code to string values. That way the only thing you need to correlate are the language codes, and that's much easier to sanity check, all the info would be in one place. After #74 is good to go this can be expressed simply as a type.

Reply:
but yea doens't need to block this